src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -376,6 +376,8 @@ public String getHiveTableName(String topic) {
         return mProperties.getString(key, null);
     }
 
+    public boolean getSkipQuboleAddPartition() { return getBoolean("secor.skip.qubole.add.partition", false); }
```

Comment:
secor has the convention that config parameters are explicitly defined in secor.common.properties, usually no java code defaulting.